docker/nginx.conf

```diff
@@ -214,6 +214,22 @@ http {
                 return 200;
             }
         }
+
+        location / {
```

Comment:
Is this thing adding MQTT or Dashflux?

Reply:
This is adding Dashflux, because now, compose and k8s share same configuration, and k8s already had Dashflux.

Reply:
I see, but PR name is confusing then.

Reply:
Point of this PR is to add MQTT to nginx for kubernetes. This change was needed as I had to share nginx configuration.